src/ig/IGExporter.ts

```diff
@@ -782,6 +782,11 @@ export class IGExporter {
     );
     instances
       .filter(instance => instance._instanceMeta.usage !== 'Inline')
+      .filter(
+        instance =>
+          // Filter out custom defintions that contain a type that has the same type as our instance type
```

Comment:
I might just be getting confused, but this comment sounds like it might be describing something different than the code does. This is filtering out any instance that has a type that is one of the custom resources defined in the package, right? Maybe the comment could be something like 
```suggestion
          // Filter out instances that have a type that has the same type as a custom resource defined in the package
```

Reply:
I'm not sure where/how this happened, but I don't see this change anymore? Should it have been included?